corehq/apps/data_cleaning/templates/data_cleaning/summary/apply_changes.html

```diff
@@ -1,8 +1,20 @@
 {% load i18n %}
 
-<p class="lead">
-  {% blocktrans %}
-    The following edits will be applied:
-  {% endblocktrans %}
-</p>
-{# todo list summary of all session changes #}
+{% if changes %}
+  <p class="lead">
+    {# prettier-ignore-start #}
```

Comment:
Curious what issues Prettier was having with this block

Reply:
it was treating `{% plural %}` as an inline tag and not part of `{% blocktrans %}`. easy mistake to mess up...django templates are tricky